omnigan/trainer.py

```diff
@@ -1606,6 +1607,18 @@ def functional_test_mode(self):
             self.exp.log_parameter("is_functional_test", True)
         atexit.register(self.del_output_path)
 
+    def compute_fire(self, x, seg_preds):
+        if seg_preds is None:
```

Comment:
we might use it with `z`, can you make this function handle a `z` input like I did for smog?

Reply:
Done.:)